.github/workflows/build.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           - 23
         # Collect coverage on latest LTS
         include:
-          - os: ubuntu-20.04
+          - os: ubuntu-latest
```

Comment:
the matrix above is already using `ubuntu-latest` so seemed ok to go with that here(?)